Doc/library/unittest.rst

```diff
@@ -531,16 +532,20 @@ Basic skipping looks like this::
            # windows specific testing code
            pass
 
+       def test_skip_me(self):
+           self.skipTest("skip")
```

Comment:
Nit, but I'd prefer not to encourage meaningless skip messages :).  How about

```suggestion
           self.skipTest("skipped from within")
```

or similar?  Even better might be to show that this can be called conditionally, something like:

```suggestion
           if not external_resource_available():
               self.skipTest("external resource not available")
           # test code that depends on the external resource
           pass
```